jetty-server/src/main/java/org/eclipse/jetty/server/AbstractNCSARequestLog.java

```diff
@@ -489,4 +496,58 @@ public String getLogTimeZone()
     {
         return _logTimeZone;
     }
+    
+
+    private String extractUser(Request request)
```

Comment:
@mat013 what happens if someone submits a request with bogus Authentication / Authorization headers (or more then header)?
You've now logged that bogus header information in the request log, in the field specifically for authenticated user IDs.